packages/front-end/tests/e2e/lab-manager/Labs_Lab_Manager.spec.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from 'playwright/test';
+
+test('01 - Create a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/labs`);
+  await page.waitForTimeout(2000);
+
+  // confirm button if enabled/visible
+  await expect(page.getByRole('button', { name: 'Create a new Lab' })).toBeHidden();
+});
+
+test('02 - Remove a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/labs`);
+  await page.waitForLoadState('networkidle');
+
+  // Check if any test lab exists
+  let hasTestLab = true;
+  try {
+    hasTestLab = await page.getByRole('row').nth(1).isVisible();
+  } catch (error) {
+    console.log('No lab not found:', error);
+  }
+
+  if (hasTestLab) {
+    // Click on the first lab row available
+    await page.getByRole('row').locator('button').nth(1).click();
+
+    // Confirm if Remove option is available on the first row
+    await expect(page.getByRole('menuitem', { name: 'Remove' })).toBeHidden();
+  }
+});
+
+test('03 - View Org Page Unsuccessfully', async ({ page, baseURL }) => {
```

Comment:
Should this read "Successfully" instead of "Unsuccessfully"?

Reply:
This one checks for Organization menu that it should not be available